UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -128,13 +121,16 @@ public float MaxHealth
 		/// <summary>
 		/// A list of all body parts of the creature
 		/// </summary>
-		public HashSet<BodyPart> BodyPartList = new HashSet<BodyPart>();
+		public List<BodyPart> BodyPartList = new List<BodyPart>();
```

Comment:
Just curious, is there something about `HashSet` that means we should avoid it, or were the collection's advantages just not used in this case?

Reply:
There were occasions where double body parts ended up getting added so I used HashSet to just prevent that

Reply:
this doesn't seem to happen anymore, I'm unaware of the cause of the bug